### PR TITLE
Migrated thumbnails for Python docs to Plotly CDN.

### PR DESCRIPTION
**Pull request by tarzzz**

### CODE REVIEW

**Comment by chriddyp** [comment]
nice! looks good @tarzzz

File: _config_dev.yml [hunk 1]
```diff
@@ -9,10 +9,11 @@ staticurl: http://localhost:4000/all_static
 exclude: ['*.Rmd','_posts/2015-09-09-matlab-reference.html','_posts/2015-08-19-plotly_js-reference.html','_posts/2015-04-05-ggplot2-index.html','_posts/2015-04-05-julia-index.html','_posts/2015-04-05-matlab-index.html','_posts/2015-04-05-matplotlib-index.html','_posts/2015-04-05-node_js-index.html','_posts/2015-04-05-plotly_js-index.html','_posts/2015-04-05-plotlyjs-function-reference.md','_posts/2015-04-05-python-index.html','_posts/2015-05-25-ipython-notebooks_gallery.html','_posts/2015-05-25-ipython-notebooks_index.html','_posts/2015-07-13-eula_index.html','_posts/2015-07-19-pandas.html','_posts/2015-07-26-index.html','_posts/2015-07-30-r-index.Rmd','_posts/2015-07-30-r-index.md','_posts/2015-08-20-research-box-index.html','_posts/ggplot2','_posts/julia','_posts/matlab','_posts/matplotlib','_posts/nodejs','_posts/pandas','_posts/plotly_js','_posts/python','_posts/r','_posts/tutorials','_posts/user_guide_matlab','_posts/user_guide_python']
 
 plotlyjs_download_url: http://cdn.plot.ly/plotly-latest.min.js
```
Comment (etpinard) [inline _config_dev.yml]: this should be `https:// ...`

the plain `http` works also, but it should be discourage as it won't work for folks working in CodePens and jsFiddles.

Comment (tarzzz) [inline _config_dev.yml]: Yep, Need to update it! :+1:

**Comment by tarzzz** [comment]
added thumbnails for R, and Python. Two of the thumbnails in Python examples (Wireframe, and networks) have some sizing issues.

**Comment by tarzzz** [comment]
@chriddyp @jackparmer I am getting the SSL Error while trying to access images.plot.ly with 'https' protocol:
image

Falling back to 'Http' for now.

**Comment by jackparmer** [comment]
Hey Tarun, Yes - images.plot.ly isn't set up for httpS - please use http
instead. Thanks!

**Comment by tarzzz** [comment]
Added CDN links for Plotly JS, matplotlib, and ipython notebook examples .. (phew)

**Comment by chriddyp** [comment]
hey @tarzzz - Can I merge this progress in? Feel free to make additional updates in subsequent PRs.

**Comment by tarzzz** [comment]
@chriddyp Yes, Please merge it, it is stable. As for performance, you are in for a surprise.. ;)

**Comment by chriddyp** [comment]
looks great @tarzzz !

**Pull request merged**